handler/src/main/java/io/netty/handler/timeout/IdleStateHandler.java

```diff
@@ -110,6 +110,13 @@ public void operationComplete(ChannelFuture future) throws Exception {
     private final long writerIdleTimeNanos;
     private final long allIdleTimeNanos;
 
+    private final IdleStateEvent FIRST_READER_IDLE_STATE_EVENT = new IdleStateEvent(IdleState.READER_IDLE, true, this);
+    private final IdleStateEvent READER_IDLE_STATE_EVENT = new IdleStateEvent(IdleState.READER_IDLE, false, this);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "READER_IDLE_STATE_EVENT" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS00116)